### PR TITLE
minor edits

### PR DESCRIPTION
**Pull request by dylanbeaudette**
A bunch of small edits for clarity, brevity, and links to additional resources.

### CODE REVIEW

**Comment by brownag** [comment]
I don't see anything wrong with this

**Review by brownag — APPROVED** [review]

File: newbook/02-data.Rmd [hunk 1]
```diff
@@ -977,8 +975,9 @@ We can compare this to what we see in the NASIS Pedon Diagnostic Features table:
 
 Any profiles that have logic errors detected are stored in `soilDB.env` `bad.pedon.ids` variable after you run `fetchNASIS`:
 
+<!-- consider eval=FALSE for less output that contains very little information -->
```
Comment (brownag) [inline newbook/02-data.Rmd:978]: I want the output in there -- please do not remove this / remove this comment

**Review by jskovlin — APPROVED** [review]
These changes look good - I approve.

File: newbook/02-data.Rmd (hunk 1 above)
Comment (dylanbeaudette) [reply newbook/02-data.Rmd:978]: OK.

**Pull request merged**